index.js

```diff
@@ -21,7 +21,35 @@ mongoose
   })
   .then(() => {
     // Run your code here, after you have insured that the connection was made
+    // Iteration 2
+    Recipe.create(data[0])
+      .then(() => {
+        Recipe.find({}, {title: 1})
```

Comment:
Using the .find here is good for being extra sure it is in the DB but might actually not be needed. The .create promise will give you back the element it created inside the parameter of its .then(). So you can you console.log it from there.